ui/package.json

```diff
@@ -1,7 +1,26 @@
 {
-  "name": "ui",
-  "version": "0.1.0",
-  "private": true,
+  "name": "numaflow-ui-test",
```

Comment:
test?

Reply:
currently, we have not created an account for numaflow on npm, so I am using my personal account, thus the test name and author, once we have the account I will update this.

Reply:
can you please open an issue to track this?

Reply:
created - https://github.com/numaproj/numaflow/issues/1585